diff --git a/Dockerfile b/Dockerfile
index c1ed88c..5bbc253 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,6 @@
 FROM python:3.7-alpine3.8
 
-ENV workdir /data
+ENV workdir=/data
 WORKDIR ${workdir}
 
 RUN mkdir -p ${workdir} && adduser python --disabled-password
diff --git a/youtube_upload/main.py b/youtube_upload/main.py
index af56e6a..de62c0f 100755
--- a/youtube_upload/main.py
+++ b/youtube_upload/main.py
@@ -35,7 +35,7 @@ from . import playlists
 
 # http://code.google.com/p/python-progressbar (>= 2.3)
 try:
-    import progressbar2
+    import progressbar
 except ImportError:
     progressbar = None
 
@@ -148,7 +148,7 @@ def upload_youtube_video(youtube, options, video_path, total_videos, index):
             "privacyStatus": ("private" if options.publish_at else options.privacy),
             "publishAt": options.publish_at,
             "license": options.license,
-
+            "selfDeclaredMadeForKids": False,
         },
         "recordingDetails": {
             "location": lib.string_to_dict(options.location),
